qa/acceptance-tests/src/test/java/io/camunda/it/client/ProcessInstanceAndElementInstanceSearchTest.java

```diff
@@ -1670,4 +1670,57 @@ void shouldQueryProcessInstancesByIncidentErrorHashCode() {
     assertThat(result.items().size()).isEqualTo(1);
     assertThat(result.items()).extracting("processDefinitionId").contains("incident_process_v2");
   }
+
+  @Test
+  void shouldQueryProcessInstanceCallHierarchy() {
+    // given
+    final long childProcessInstanceKey =
+        camundaClient
+            .newProcessInstanceSearchRequest()
+            .filter(f -> f.processDefinitionId("child_process_v1"))
+            .send()
+            .join()
+            .items()
+            .getFirst()
+            .getProcessInstanceKey();
+
+    // when
+    final var result =
+        camundaClient
+            .newProcessInstanceGetCallHierarchyRequest(childProcessInstanceKey)
+            .send()
+            .join();
+
+    // then
+    assertThat(result).isNotNull();
+    assertThat(result).hasSize(2);
+    assertThat(result)
```

Comment:
🙃 Just a nitpick: can these calls be chained directly? They all use the same base assertion, right?

```suggestion
    assertThat(result)
        .isNotNull()
        .hasSize(2)
```